libx52/x52_core.c

```diff
@@ -55,7 +55,11 @@ libx52_device* libx52_init(void)
 
     rc = libusb_init(&(x52_dev->ctx));
     if (rc) {
+#if defined(__linux__)
```

Comment:
This is OK for now, although I should really clean this up to translate a libusb error to a standard error.